common/changes/@itwin/appui-react/952-allow-to-enable-multiple-snaps_2024-08-09-18-12.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/appui-react",
+      "comment": "We are now able to activate multiple snap modes at once. We can also choose what snap modes are available in the app via SnapModeField  snapModes props. ",
```

Comment:
Could you reword this to avoid `we are/can`, i.e.: "Added XYZ to `SnapModeField` component", "Multiple snap modes can be activated when using `SnapModeField` component.